components/mpas-seaice/src/shared/mpas_seaice_icepack.F

```diff
@@ -12141,7 +12162,7 @@ subroutine init_icepack_package_configs(domain)
          hfrazilmin_in           = seaiceFrazilMinimumThickness, &
          !floediam_in             = , &
          depressT_in             = seaiceMeltingTemperatureDepression, &
-         dragio_in               = seaiceIceOceanDragCoefficient, & ! note calc_dragio not implemented
```

Comment:
If seaiceIceOceanDragCoefficient and seaiceSnowConductivity are no longer used anywhere else in MPAS-SI, then I recommend removing them completely to avoid confusion, including from mpas_seaice_constants.F.  They are only in there because there wasn't a config for them!

Reply:
They are still used by ColPkg, so they need to stay.  I started changing ColPkg, but you requested that I didn't make that change there, so we still need them in there.